.github/workflows/issue.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow posts an automated comment on every new issue
+# https://github.com/marketplace/actions/create-or-update-comment (https://github.com/peter-evans/create-or-update-comment)
+
+name: Automatic Comment
+on:
+  issues:
+    types: [opened]
+permissions:
+  contents: read
+
+jobs:
+  comment:
+    permissions:
+      issues: write  # for peter-evans/create-or-update-comment to create or update comment
+      pull-requests: write  # for peter-evans/create-or-update-comment to create or update comment
+    name: Comment
+    runs-on: ubuntu-latest
+    steps:
+      - name: Automatic Comment
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          issue-number: ${{ github.event.issue.number }}
+          body: |
+            Thanks for the issue, our team will look into it as soon as possible! If you would like to work on this issue, please wait for us to decide if it's ready. We will remove the "needs triage" label when it's ready to be worked on, and you can claim it then.
```

Comment:
wording (blocking): We should mention that the issue is available once the `needs triage` label has been removed.

Related to this, it'd be nice to update the `.take` GitHub action to block assignment if certain labels are present on the issue, e.g. `needs triage`

Reply:
I guess I'll have to make the second change in https://github.com/bdougie/take-action/blob/main/action.yml, I can block assignment but I don't think I can send a message about why it was blocked if I do this (if: part):

```
 take-issue:
    name: Disable take issue
    runs-on: ubuntu-latest
    timeout-minutes: 10
    if: "!contains(github.event.issue.labels.*.name, 'needs triage')"
    steps:
      - name: take an issue
        uses: bdougie/take-action@main
        with:
          issueCurrentlyAssignedMessage: Thanks for being interested in this issue. It looks like this ticket is already assigned to a contributor.
          token: ${{ secrets.GITHUB_TOKEN }}
```